lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu'
+require_relative 'twilio_sender'
+
+class Takeaway
+
+  attr_reader :current_order
+
+  def initialize(menu = Menu, twilio = TwilioSender)
+    @menu = menu.new
+    @twilio = twilio.new
+    @current_order = []
+  end
+
+  def add_to_order(dish)
+    @current_order << dish_finder(dish)
```

Comment:
All of your methods are really well abstracted: almost all around 3 lines max and descriptive names so that it is easy to follow the logic.